packages/react/src/elements/components/Notification/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from "react";
```

Comment:
Please rename this file to `Notification.js`, and add a new `index.js` file that does this:

```
export { default } from './Notifications;
```

The convention separates the choice of what all to export from the implementation, and it lets you `Command+T` for the component name.

Reply:
sounds good!